bin/next-build-tools.js

```diff
@@ -228,6 +228,7 @@ program
 	.option('--procfile', 'Runs all processes specified in the Procfile')
 	.option('-s, --script <file>', 'Runs a single javascript file')
 	.option('--subargs [subargs]', 'Sub arguments to pass to a single script', /^\[.+]$/)
+	.option('--no-nodemon', 'Do not run through nodemon')
```

Comment:
hmm, i think we normally use `skip` as in `--skip-nodemon` for this… but sounds a little awkward so don't really mind if you keep as is